### DISCUSSION TITLE
How do I enter my license key ?

### DISCUSSION THREAD

**Discussion by francescofoti** [twinbasic/twinbasic#504] [Q&A] [answered]
I may have missed (or dismissed, can't remember) the dialog that allows me to enter my license key.
How / where can I appropriately enter my license key now ? (should I remove/add the TB extension again?)

**Comment by WaynePhillipsEA** [comment, marked as answer]
Hi, no problem - just press F1 then type 'twinBASIC: Enter Licence Key' and then you should be prompted to enter it.
